aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -2099,7 +2098,7 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
 
     @SuppressLint("InflateParams")
     fun showBlockEditorDialog(unknownHtmlSpan: UnknownHtmlSpan, html: String = "") {
-        val builder = AlertDialog.Builder(context)
+        val builder = AlertDialog.Builder(context, R.style.ResizableDialogTheme)
```

Comment:
Praise (❤️): Thanks for taking the time to resolve this suppression with the most appropriate fix for this specific case, you rock! 🥇